os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -681,30 +704,87 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name, u32
 	ptask->task = NULL;
 	ptask->task_name = NULL;
 	return _FAIL;
+#else
+	int func_addr, ctx_addr;
+	pid_t pid;
+	char str_func_addr[8+1];
```

Comment:
Are there  any reason to use `8+1` instead of `9`?

Reply:
Modified.